### DISCUSSION TITLE
找到了另一个日V常用的评论栏软件

### DISCUSSION THREAD

**Discussion by xfgryujk** [xfgryujk/blivechat#202] [通用]
https://onecomme.com/

**Comment by xfgryujk** [comment]
翻译功能用的是另一个软件[ゆかコネNeo](https://nmori.github.io/yncneo-Docs/download/)。这个也支持语音识别和翻译到其他语言，可以通过浏览器源导入OBS

语音识别默认是用浏览器的API [SpeechRecognition](https://developer.mozilla.org/en-US/docs/Web/API/SpeechRecognition)，另外也可以选择用[Vosk](https://github.com/alphacep/vosk-api)或[Whisper](https://github.com/tyapa0/YukariWhisper)离线识别

翻译引擎虽然也支持腾讯和百度翻译，但是日本人一般不会配置API key，所以一般用谷歌翻译，效果你懂的。也许可以做个插件接入[Sakura](https://github.com/SakuraLLM/Sakura-13B-Galgame)。我测试的时候翻译过程是同步的，等待翻译的时候不会显示字幕，这个体验很差，不知道是不是我配置错了

**Reply by xfgryujk** [reply]
好像开启节约翻译API使用后就会变成同步的 https://nmori.github.io/yncneo-Docs/startup/startup_option/

**Comment by SSSSSSSan** [comment]
（这个我认识的台V也在用
